ssm/invocation/runner.go

```diff
@@ -1,111 +1,67 @@
 package invocation
 
 import (
-	"sync"
+	"fmt"
 	"time"
 
-	"github.com/aws/aws-sdk-go/aws"
-	"github.com/aws/aws-sdk-go/aws/awserr"
 	"github.com/aws/aws-sdk-go/service/ssm"
 	"github.com/aws/aws-sdk-go/service/ssm/ssmiface"
-	log "github.com/sirupsen/logrus"
 )
 
 // RunSSMCommand uses an SSM session, pre-defined SSM document parameters, the dry run flag, and any number of instance IDs and executes the given command
 // using the AWS-RunShellScript SSM document. It returns an *ssm.SendCommandOutput object, which contains the execution ID of the command, which we use to
 // check the progress/status of the invocation.
-func RunSSMCommand(session ssmiface.SSMAPI, params *RunShellScriptParameters, dryRunFlag bool, resultChan chan *ssm.SendCommandOutput, errChan chan error, instanceID ...string) {
-	var err error
-	var output *ssm.SendCommandOutput
-
-	ssmCommandInput := &ssm.SendCommandInput{
-		DocumentName: aws.String("AWS-RunShellScript"),
-		InstanceIds:  aws.StringSlice(instanceID),
-		Parameters:   *params}
+func RunSSMCommand(session ssmiface.SSMAPI, input *ssm.SendCommandInput, dryRunFlag bool) (scOutput *ssm.SendCommandOutput, err error) {
```

Comment:
Similar comment here. The object passed here is a struct that fulfills the ssm client interface. For consistency 'session' should be changed to 'client'.